### ISSUE TITLE
第六轮评估：序号141-150

### ISSUE THREAD

**Issue by mrcangye** [PaddlePaddle/docs#5212]
### 文档中英文链接

1. paddle.nn.Sigmoid
    [中：https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/api/paddle/nn/Sigmoid_cn.html#sigmoid](https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/api/paddle/nn/Sigmoid_cn.html#sigmoid)
    [英：https://www.paddlepaddle.org.cn/documentation/docs/en/develop/api/paddle/nn/Sigmoid_en.html#sigmoid](https://www.paddlepaddle.org.cn/documentation/docs/en/develop/api/paddle/nn/Sigmoid_en.html#sigmoid)
    存在问题：公式问题，PR修复

2. paddle.nn.functional.sigmoid
    [中：https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/api/paddle/nn/functional/sigmoid_cn.html#sigmoid](https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/api/paddle/nn/functional/sigmoid_cn.html#sigmoid)
    [英：https://www.paddlepaddle.org.cn/documentation/docs/en/develop/api/paddle/nn/functional/sigmoid_en.html](https://www.paddlepaddle.org.cn/documentation/docs/en/develop/api/paddle/nn/functional/sigmoid_en.html)
    存在问题：源码跳转链接不正确，英文公式无法修改

3. paddle.nn.Silu
    [中：https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/api/paddle/nn/Silu_cn.html#silu](https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/api/paddle/nn/Silu_cn.html#silu)
    [英：https://www.paddlepaddle.org.cn/documentation/docs/en/develop/api/paddle/nn/Silu_en.html](https://www.paddlepaddle.org.cn/documentation/docs/en/develop/api/paddle/nn/Silu_en.html)
    存在问题：class paddle.nn.Silu英文页面报错

4. paddle.nn.functional.silu

    [中：https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/api/paddle/nn/functional/silu_cn.html#silu](https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/api/paddle/nn/functional/silu_cn.html#silu)
    [英：https://www.paddlepaddle.org.cn/documentation/docs/en/develop/api/paddle/nn/functional/silu_en.html](https://www.paddlepaddle.org.cn/documentation/docs/en/develop/api/paddle/nn/functional/silu_en.html)
    存在问题：公式问题，PR修复

5. paddle.nn.SmoothL1Loss

    [中：https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/api/paddle/nn/SmoothL1Loss_cn.html#smoothl1loss](https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/api/paddle/nn/SmoothL1Loss_cn.html#smoothl1loss)
    [英：https://www.paddlepaddle.org.cn/documentation/docs/en/develop/api/paddle/nn/SmoothL1Loss_en.html](https://www.paddlepaddle.org.cn/documentation/docs/en/develop/api/paddle/nn/SmoothL1Loss_en.html)
    存在问题：出现op用语，zi公式未能成功显示，代码示例没有结果输出，PR修复

6. paddle.nn.functional.smooth_l1_loss

    [中：https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/api/paddle/nn/functional/smooth_l1_loss_cn.html#smooth-l1-loss](https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/api/paddle/nn/functional/smooth_l1_loss_cn.html#smooth-l1-loss)
    [英：https://www.paddlepaddle.org.cn/documentation/docs/en/develop/api/paddle/nn/functional/smooth_l1_loss_en.html](https://www.paddlepaddle.org.cn/documentation/docs/en/develop/api/paddle/nn/functional/smooth_l1_loss_en.html)
    存在问题：出现op用语，zi公式未能成功显示，代码示例没有结果输出，PR修复

7. paddle.nn.Softmax

    [中：https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/api/paddle/nn/Softmax_cn.html#softmax](https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/api/paddle/nn/Softmax_cn.html#softmax)
    [英：https://www.paddlepaddle.org.cn/documentation/docs/en/develop/api/paddle/nn/Softmax_en.html](https://www.paddlepaddle.org.cn/documentation/docs/en/develop/api/paddle/nn/Softmax_en.html)
    存在问题：出现op用语，PR修复

8. paddle.nn.functional.softmax

    [中：https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/api/paddle/nn/functional/softmax_cn.html#softmax](https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/api/paddle/nn/functional/softmax_cn.html#softmax)
    [英：https://www.paddlepaddle.org.cn/documentation/docs/en/develop/api/paddle/nn/functional/softmax_en.html](https://www.paddlepaddle.org.cn/documentation/docs/en/develop/api/paddle/nn/functional/softmax_en.html)
    存在问题：出现op用语，PR修复

9. paddle.nn.Softplus

    [中：https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/api/paddle/nn/Softplus_cn.html#softplus](https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/api/paddle/nn/Softplus_cn.html#softplus)
    [英：https://www.paddlepaddle.org.cn/documentation/docs/en/develop/api/paddle/nn/Softplus_en.html](https://www.paddlepaddle.org.cn/documentation/docs/en/develop/api/paddle/nn/Softplus_en.html)
    存在问题：公式问题，PR修复

10. paddle.nn.functional.softplus

    [中：https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/api/paddle/nn/functional/softplus_cn.html#softplus](https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/api/paddle/nn/functional/softplus_cn.html#softplus)
    [英：https://www.paddlepaddle.org.cn/documentation/docs/en/develop/api/paddle/nn/functional/softplus_en.html](https://www.paddlepaddle.org.cn/documentation/docs/en/develop/api/paddle/nn/functional/softplus_en.html)
    存在问题：公式问题，PR修复

### 红线问题

1. paddle.nn.functional.sigmoid
    [中：https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/api/paddle/nn/functional/sigmoid_cn.html#sigmoid](https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/api/paddle/nn/functional/sigmoid_cn.html#sigmoid)
    [英：https://www.paddlepaddle.org.cn/documentation/docs/en/develop/api/paddle/nn/functional/sigmoid_en.html](https://www.paddlepaddle.org.cn/documentation/docs/en/develop/api/paddle/nn/functional/sigmoid_en.html)
    存在问题：源码跳转链接不正确，英文公式无法修改

2. paddle.nn.Silu
    [中：https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/api/paddle/nn/Silu_cn.html#silu](https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/api/paddle/nn/Silu_cn.html#silu)
    [英：https://www.paddlepaddle.org.cn/documentation/docs/en/develop/api/paddle/nn/Silu_en.html](https://www.paddlepaddle.org.cn/documentation/docs/en/develop/api/paddle/nn/Silu_en.html)
    存在问题：class paddle.nn.Silu英文页面报错

### 主观评价

PR链接：
https://github.com/PaddlePaddle/docs/pull/5211
